ads-with-uniregistry.rst

```diff
@@ -0,0 +1,73 @@
+.. post:: 
+   :tags: advertising, sustainability, draft
+
+Uniregistry sponsors Read the Docs and Open Source
+==================================================
+
+Today we're excited to announce an important sponsorship partner in our `Ethical Advertising`_ campaign: `Uniregistry`_.  Our goal with our ethical advertising program is to provide important funding for open source, and show that it can be done ethically.
+
+Domain registration was identified early as a natural partner to our program, because it sits in the stack of necessary infrastructure for all of us that work on making the internet. We wanted the right partner, because historically we feel that domain registration companies left room for improvement in the customer experience. This is post covers a few of the criteria we used to reach the conclusion to partner with Uniregistry.
+
+To not bury the lede,
+we're offering 25% off first year registrations and/or transfers into Uniregistry,
+using the code ``DOCS25``.
+We hope that you'll enjoy getting started with the service,
+and find it as useful.
+`Sign up`_ now to get started.
```

Comment:
* "To not bury the lede" is more overly explicit language. This can be removed
* "We're offering" is misleading. I generally see something like "Uniregistry is offering Read the Docs users ..."
* "We hope .. you'll enjoy the service" is also misleading a bit, "we" are not the ones offering the service. There is probably a better way of saying this.